src/pages/bridge/Send/useEstimateSendTransaction.tsx

```diff
@@ -25,28 +25,32 @@ export function useEstimateSendTransaction(props) {
   }, [networksAndSigners])
 
   const depositETH = async () => {
-    const fee = await getPriceFee(selectedToken, fromNetwork.isL1)
-    return instance["depositETH(uint256,uint256)"].estimateGas(minimumAmount, GAS_LIMIT.DEPOSIT_ETH, {
+    const gasLimit = await getGasLimit()
```

Comment:
could the three lines of code be replaced with `const fee = await getPriceFee()`, it seems like they are equivalent.

Reply:
I am wondering if it would be more appropriate for `usePriceFee` to return `gasFee` and `gasLimit`